app/components/agenda/agenda-detail/sidebar-item.hbs

```diff
@@ -41,7 +41,7 @@
               {{#if this.renderDetails}}
                 {{#if (await @agendaitem.checkAdded)}}
                   <span class="added-tag vlc-agenda-meta__recently-added">
```

Comment:
Wat is de "added-tag" klasse? Waar dient dit voor? Precies geen namespace.

Reply:
Ik check dit morgen direct even 👍

Reply:
Die classname had geen nut. Vermoedelijk iets van al-verwijderde code of van een oude package?

Reply:
Geen idee wat dit is.